pyresample/geometry.py

```diff
@@ -2595,6 +2633,17 @@ def get_geostationary_bounding_box(geos_area, nb_points=50):
     x = np.clip(np.concatenate([x, x[::-1]]), min(ll_x, ur_x), max(ll_x, ur_x))
     y = np.clip(np.concatenate([y, -y]), min(ll_y, ur_y), max(ll_y, ur_y))
 
+    return x, y
+
+
+def get_geostationary_bounding_box(geos_area, nb_points=50):
```

Comment:
Should this function include `_in_lonlats` similar to `in_proj_coords` like the other geos bounding box function? Also are these two functions used outside this module? Can they be private?

Reply:
Added the `in_lonlats` version.

These functions are used outside of the module.